Sync status

Sirmorrison opened the pull request:
### Resolves

Issue #164 

### What's new

This PR improves the Overview Page.

### Relevant screenshots or logs

<img width="796" alt="image" src="https://user-images.githubusercontent.com/27733432/86128323-23d43200-bad9-11ea-828e-c335d6089524.png">


<img width="796" alt="image" src="https://user-images.githubusercontent.com/27733432/86128397-39495c00-bad9-11ea-86ce-f98ebe0b9b0d.png">

<img width="798" alt="image" src="https://user-images.githubusercontent.com/27733432/86128474-4fefb300-bad9-11ea-92f6-b9b48a4f3cd8.png">

Sirmorrison committed:
add sync status icons
change dynamic btn color

Sirmorrison committed:
improve page layout
change dynamic sync btn color to static
add online/offline status indicator icon
add line seperator
Add view all tx btn

Sirmorrison committed:
update sync block to mobile ui

Sirmorrison committed:
change the color of the wallet sync boxes to gray
change syncing status icon to be same as mobile

Sirmorrison committed:
fix image icon size issues and adjust icon color

Sirmorrison committed:
uniform spacing for transactionsection

Sirmorrison committed:
rebase PR

Sirmorrison committed:
fix lint

Sirmorrison committed:
center align tx table.

Sirmorrison committed:
fix overview txn table excess padding issue

Sirmorrison committed:
fix txn table excess padding on tx page

oshorefueled commented on ui/overview_page.go:
```diff
@@ -227,82 +262,82 @@ func (page *overviewPage) recentTransactionsColumn(c pageCommon) {
+			})
+		}),
+		layout.Rigid(func() {
+			gtx.Constraints.Width.Min = gtx.Px(unit.Dp(100))
```
You should define this in dimension.go

oshorefueled commented on ui/overview_page.go:
```diff
@@ -227,82 +262,82 @@ func (page *overviewPage) recentTransactionsColumn(c pageCommon) {
-					}),
-				)
+		layout.Rigid(func() {
+			gtx.Constraints.Width.Min = gtx.Px(unit.Dp(100))
```
You should define this in dimension.go

oshorefueled commented on ui/overview_page.go:
```diff
@@ -227,82 +262,82 @@ func (page *overviewPage) recentTransactionsColumn(c pageCommon) {
 			})
 		}),
 		layout.Rigid(func() {
+			gtx.Constraints.Width.Min = gtx.Px(unit.Dp(100))
```
You should define this in dimension.go, there shouldn't be hard coded dimensions in your code.

oshorefueled commented on ui/overview_page.go:
```diff
@@ -382,27 +481,65 @@ func (page *overviewPage) endToEndRow(inset layout.Inset, leftLabel, rightLabel
+					layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+						layout.Rigid(func() {
+							layout.Inset{Top: unit.Dp(6), Right: unit.Dp(20)}.Layout(gtx, func() {
+								page.walletStatusIcon.Layout(gtx, unit.Dp(10))
```
remove hard coded dimensions. Dimensions defined in dimension.go are standardized to a 5 unit difference. So instead of 6, you can use 5 and instead of 8, you can use 10. You can define a unique dimension that don't follow the standardized unit difference only when it's absolutely necessary.

oshorefueled commented on ui/transactions_page.go:
```diff
@@ -187,24 +188,29 @@ func (page *transactionsPage) txnRowHeader(common *pageCommon) {
 	layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
-		layout.Flexed(page.rowDirectionWidth, func() {
+		layout.Rigid(func() {
+			gtx.Constraints.Width.Min = gtx.Px(unit.Dp(70))
```
dimensions should be defined in dimensions.go

Sirmorrison committed:
move hard coded dimensions to dimension.go
centralize tx page trx table on large screen
adjust color on overview page sunc section

Sirmorrison committed:
remove fill from card widget
left align tx table on txn page

Sirmorrison committed:
rebase pr
